guides/data_science/machine_learning/Tensorflow.md

```diff
@@ -0,0 +1,60 @@
+# Introduction
+
+[**TensorFlow**][1] is an open source software library for high performance numerical computation & machine learning applications such as [neural networks][2]. Its flexible architecture allows easy deployment of computation across a variety of platforms viz. CPUs, GPUs, TPUs.
+
+![](https://i0.wp.com/pbiswas101.files.wordpress.com/2018/07/tfjs.jpg?ssl=1&w=450)
+
+ Originally developed by the [Google Brain][3] team for internal **Google** use, until it was released under the Apache 2.0 open source license on November 9, 2015.
+
+The name "**TensorFlow**" has been adopted based on the multidimensional array ( [*tensors*][4] ) which are useful for data-driven operation such as [Image Processing][5], [Natural Language Processing][6], [Deep Learning][7], etc.
+
+> Google announced dedicated TensorFlow support for both mobile devices as well as web browsers!
+
+- [**TensorFlow Lite**][8], a software stack specifically for Android Development
+- [**TensorFlow.js**][9], a library for training and deploying ML models on the web
+
+## TensorFlow Lite
+
+TensorFlow Lite is TensorFlow’s lightweight solution for mobile & embedded devices. It enables on-device machine learning inference with low latency and a small binary size. Here, is the [documentation guide][10]!
+
+![](https://i1.wp.com/pbiswas101.files.wordpress.com/2018/07/tensorflow-lite.jpg?ssl=1&w=350)
+
+TensorFlow Lite provides an interface to leverage hardware acceleration via the [Android Neural Networks API][11], available on Android 8.1 (Oreo) and higher.
+
+## TensorFlow.js
+
+Tensorflow.js is a JavaScript library for training and deploying ML models in the browser and on Node.js. It is built on top of low level API [deeplearn.js][12]
+
+![](https://i0.wp.com/pbiswas101.files.wordpress.com/2018/07/tensorflow-js.png?ssl=1&w=450)
+
+Tensorflow.js enables the following advantage without needing to install additional drivers since it is up & running on the browser:
+
+- Develop ML with JavaScript
+- Run Existing Models
+- Retrain Existing Models
+
+------------
+
+###### Open Source. Get Started
+
+TensorFlow is an wonderful library. In order to build awesome models using this library you need to have prior programming experience using **Python**. Check out the [official site][13] & you can also contribute in it's official [GitHub repo][14].

```

Comment:
Add this stuff too:
- Why is it better/worse than other DL libraries
- Some cool research papers from arXiv which have used tensorflow(optional to include)
- Tensorboard